mlflow/entities/model_registry/model_version_stages.py

```diff
@@ -9,7 +9,7 @@
 STAGE_DELETED_INTERNAL = "Deleted_Internal"
 
 ALL_STAGES = [STAGE_NONE, STAGE_STAGING, STAGE_PRODUCTION, STAGE_ARCHIVED]
-DEFAULT_STAGES_FOR_GET_LATEST_VERSIONS = [STAGE_STAGING, STAGE_PRODUCTION]
+DEFAULT_STAGES_FOR_GET_LATEST_VERSIONS = [STAGE_NONE, STAGE_STAGING, STAGE_PRODUCTION]
```

Comment:
@mparkhe and I discussed this and `NONE` was excluded intentionally. Should we change the behavior?

Reply:
If y'all had that discussion then I'll change it back. 

The docs should be updated to reflect this behavior. 
```
Latest version models for each requests stage. Only return models with current READY status. If no stages provided, returns the latest version for each stage, including "None".
```

https://www.mlflow.org/docs/latest/rest-api.html#get-latest-modelversions